about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css" />
+    <link href="https://fonts.googleapis.com/css?family=lato" rel="stylesheet"/>
+    <title>Capstone Project 1</title>
+  </head>
+  <body>
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="./about.html" class="navigate"> About </a>
+        <a href="#contact" class="navigate"> Contact </a>
+        <a href="#previous-summits" class="navigate">Summits </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
+    <header>
+      <div class="social-media">
+        <nav class="social-nav">
+          <a href="https://facebook.com/emem.ekpo"
+            ><img src="./images/facebook-icon.png" alt="facebook"
+          /></a>
+          <a href="https://twitter.com/ememcookey"
+            ><img src="./images/twitter-icon.png" alt="twitter"
+          /></a>
+          <a href="#">English</a>
+          <a href="https://github.com/ecekpo">My page</a>
+          <a href="#">Logout</a>
+        </nav>
+      </div>
+      <div class="navbar">
+        <div class="navbar-logo-div">
+          <a href="./index.html"
+            ><img
+              class="summit-logo"
+              src="./images/DGEELOGO1.png"
+              alt="Dgeeslogo"
+          /></a>
+        </div>
+
+        <nav class="navigation">
+          <a href="./index.html">Home</a>
+          <a href="#contact">Contact</a>
+          <a href="#previous-summits">Summits</a>
+          <a href="#sponsor">Sponsor</a>
+          <button class="dgees-btn" type="button">D'GEES Campaign</button>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="about-box">
+        <button type="button" class="hamburger">
+          <img src="./images/menu.png" alt="menu icon" />
+        </button>
+        <p class="hello">"Hello! INFOTECH WORLD"</p>
+        <h1 class="heading">
+          D'GEES INFOTECH <br />
+          GLOBAL SYSTEMS SUMMIT 2022
+        </h1>
+        <section class="about-intro">
+          <h2 class="mission">
+            The global summit brings together IT experts, academians both foreign and local speakers to expose the technologies that are changing the world to the participants. 
+          </h2>
+          <p class="contact" id="contact">
+            You can contact us through our email for further enquries<br />
+            about D'GEES Summit 2022 <br /><span class="email"
+              >ememekpo@yahoo.com</span
+            >
+          </p>
+        </section>
+      </div>
+      <section class="logo-details">
```

Comment:
Kindly implement a working mobile menu using inspiration from what you learned in this module when you were creating an interactive mobile menu for your Portfolio Website.

![image](https://user-images.githubusercontent.com/38088818/176731245-ed49c05a-7371-44bb-be3d-582e9cfc0b71.png)

Project Requirements:

![image](https://user-images.githubusercontent.com/38088818/176734348-d06119b2-dfa2-45f1-a9e6-45b746df31ff.png)

- I should be able to open and close the mobile menu with an (X) button.
- I should be able to navigate to the about page and back to the home page using the mobile menu.
- The menu should disappear when I click on an item.
- The menu background should cover the whole mobile screen when active.

You have learned all of this during this module. You got this 💪🏾 
Otherwise, everything else is perfect. Well done!!! 👍🏾